delta-app/src/main/java/io/cdap/delta/store/DraftService.java

```diff
@@ -229,16 +231,28 @@ public TableAssessmentResponse assessTable(DraftId draftId, Configurer configure
 
     Map<String, TableTransformation> tableLevelTransformations =
       TransformationUtil.getTableLevelTransformations(deltaConfig);
-    List<Transformation> transformations = TransformationUtil.loadTransformations(configurer,
-                                                                                  tableLevelTransformations,
-                                                                                  selectedTable);
+    Map<String, List<Transformation>> transformations = new HashMap<>();
+    List<Problem> transformationErrors = new ArrayList<>();
+    try {
+      transformations = TransformationUtil.loadTransformations(configurer, tableLevelTransformations, selectedTable);
+    } catch (TransformationException e) {
+      transformationErrors.add(new Problem("Transformation Loading failed",
+                               String.format("Failed to load transformations for the table : %s and column : %s " +
+                               "with error : %s", e.getTableName(), e.getColumnName(), e.getMessage()),
+                               "Please ensure the applied transformation's plugin is uploaded'",
+                               ""));
+    }
```

Comment:
shall we return here ? if any of the transformation fail to be loaded, we don't need to continue

Reply:
yeah makes sense. Addressed, PTAL